docs/src/examples/components/Tree/Types/TreeExample.shorthand.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { Tree } from '@stardust-ui/react'
+const treeData = [
+  {
+    title: 'one',
+    subtree: [
+      {
+        title: 'one one',
+        subtree: [
+          {
+            title: 'one one one',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    title: ' two',
+    subtree: [
+      {
+        title: 'two one',
+      },
+    ],
+  },
+]
+
+const TreeExampleShorthand = () => <Tree treedata={treeData} />
```

Comment:
Did you speak with Levi about this? I think that we should use `items` instead of `treeData` there.

```suggestion
const TreeExampleShorthand = () => <Tree items={items} />
```

Reply:
Yes Levi had a look that day. @levithomason What are your inputs?

Reply:
I'm not Levi, but I am 100% sure that we will not use `treedata` as a prop name.